eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessor.java

```diff
@@ -48,68 +53,78 @@ public SubSnpNoHgvsToDbsnpVariantsWrapperProcessor() {
     @Override
     public DbsnpVariantsWrapper process(SubSnpNoHgvs subSnpNoHgvs) throws Exception {
         DbsnpVariantsWrapper dbsnpVariantsWrapper = new DbsnpVariantsWrapper();
-        List<DbsnpSubmittedVariantEntity> variants = new ArrayList<>();
+        List<DbsnpSubmittedVariantEntity> submittedVariants = new ArrayList<>();
         List<DbsnpSubmittedVariantOperationEntity> operations = new ArrayList<>();
         DbsnpClusteredVariantEntity clusteredVariant = subSnpNoHgvsToClusteredVariantProcessor.process(subSnpNoHgvs);
 
         List<String> alternateAlleles = subSnpNoHgvs.getAlternateAllelesInForwardStrand();
         for (String alternateAllele : alternateAlleles) {
             SubmittedVariant variant = subSnpNoHgvsToSubmittedVariant(subSnpNoHgvs, alternateAllele);
             if (!variant.isAllelesMatch()) {
-                decluster(subSnpNoHgvs.getSsId(), variant, operations);
+                decluster(subSnpNoHgvs.getSsId(), variant, operations,
+                          "None of the variant alleles match the reference allele");
```

Comment:
I think it is important that we still state the variant was declustered, like you did before:

> Declustered: None of the variant alleles match the reference allele